doc/datasetManager.md

```diff
@@ -274,6 +274,131 @@ webApp:
 
 </details>
 
+### Category details
+
+In category accordions, you can open a category details dialog that can display into a Table component, some twingraph dataset data.
+These data are fetched from a Cypher request sent to the twingraph dataset.
+The table is an expression of the twingraph dataset
+The table is read only.
```

Comment:
if I'm not mistaken _data_ is uncountable :slightly_smiling_face:  Also, we could simplify this paragraph : 
- the first phrase to _The data is fetched from twingraph dataset with a Cypher query_ 
- not sure we need the second one
:slightly_smiling_face:

Reply:
Indeed but it was precision required by the PO.